### PR TITLE
Update release docs with Vivid dependency.

### PR DESCRIPTION
**Pull request by kadams54**
Also call out merging changes to the bzr precise branch.

### CODE REVIEW

**Comment by kadams54** [comment]
Changes, rendered, can be viewed here: https://github.com/kadams54/juju-gui/blob/updated-release-docs/docs/process.rst#checklist-for-making-a-stable-release

File: docs/process.rst [hunk 1]
```diff
@@ -147,6 +147,9 @@ Making Releases
 Checklist for Making a Stable Release
 -------------------------------------
 
+The release should be built in Vivid or later. Earlier versions (Trusty,
+Utopic) will run into problems with running the functional tests.
```
Comment (hatched) [inline docs/process.rst]: I'd prefer this documentation explain why vivid is required instead of just saying that it's required.

Comment (kadams54) [inline docs/process.rst]: I thought "…will run into problems running the functional tests…" was a sufficient explanation.

Comment (hatched) [inline docs/process.rst]: That doesn't answer the 'why', there are a lot of functional tests (5 whys)

Comment (kadams54) [inline docs/process.rst]: I added more detail in the "You need Vivid" note just before the functional tests.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2665/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2666/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2667/
Test PASSed.

**Comment by bac** [comment]
Just to be thorough I'm running the tests in utopic.  Will report back when they are done with the failure.  But the changes are :+1: .

**Comment by mitechie** [comment]
The SSL context was added in here:
http://bazaar.launchpad.net/~juju-gui-charmers/charms/precise/juju-gui/trunk/revision/60.119.1

This was around the bundlelib work for uncomitteed bundles. It was the first time that the GUI talked to the charm when in a demo mode. Previously that was all handled via the fake backend.

Python, pre 2.7.9 didn't actually verify SSL certs. It just checked they matched without the full upstream check. In Vivid, a version of Python landed that did do those checks. That means the SSL connection that is there would fail. This code works around that issue for testing. However, that new Python code isn't available in older versions of Python. 

This means, that to run the TEST code, you need to have vivid. It's for testing and development and not a deal killer. Folks should have a machine on vivid anyway. It's been out some time and the rule is supposed to be that internal folks are on the latest release once it hits frozen. 

I hope that helps explain the history of this as it seems to have caused some consternation among folks. It's been this way since May. Vivid came out in April. We added bundlelib support in the end of April. All the parts came together then.

**Comment by mitechie** [comment]
:+1: but feel free to commandeer anything in my note above that helps with the clarity issue Jeff is referring to.

**Comment by kadams54** [comment]
Thanks all! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/938

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2672/
Test PASSed.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/939

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/941

**Pull request merged**